articles/get-started-jupyter-notebook.md

```diff
@@ -85,15 +85,15 @@ Some things to note:
 - **5th cell**: Sets the target and submits the job. 
 - **6th and 7th cells**: Plots and displays the result.
 
-Looking at the histogram, you may notice that the program returned 0 every time, which is not very random. This is because the notebook was prepopulated to use the **Honeywell API Validator**, *honeywell.hqs-lt-s1-apival*. Using the API Validator ensures that your code will run successfully on Quantinuum hardware, but also returns 0 for every quantum measurement. 
+Looking at the histogram, you may notice that the program returned 0 every time, which is not very random. This is because the notebook was prepopulated to use the **Quantinuum Syntax Checker**, *quantinuum.hqs-lt-s1-apival*. Using the API Validator ensures that your code will run successfully on Quantinuum hardware, but also returns 0 for every quantum measurement. 
```

Comment:
```suggestion
Looking at the histogram, you may notice that the program returned 0 every time, which is not very random. This is because the notebook was prepopulated to use the **Quantinuum Syntax Checker**, *quantinuum.hqs-lt-s1-apival*. Using the Syntax Checker ensures that your code will run successfully on Quantinuum hardware, but also returns 0 for every quantum measurement. 
```